src/assets/google-script/google-script.ts

```
@@ -2,50 +1,150 @@
+        );
+
+        resolve();
+      } catch (error) {
+        this.mapReadySubject.next(false);
+        this.scriptLoaded = false;
+        this.currentLanguage = null;
+        reject(error);
+      } finally {
+        this.loadMutex = null;
+      }
+    });
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)